boards/components/src/atecc508a.rs

```diff
@@ -50,22 +57,25 @@ impl<I: 'static + i2c::I2CMaster<'static>> Atecc508aComponent<I> {
 impl<I: 'static + i2c::I2CMaster<'static>> Component for Atecc508aComponent<I> {
     type StaticInput = (
         &'static mut MaybeUninit<I2CDevice<'static, I>>,
-        &'static mut MaybeUninit<[u8; 70]>,
+        &'static mut MaybeUninit<[u8; 72]>,
```

Comment:
Is there any meaning associated with these magic numbers? Can we document this? I'm guessing they hold entropy and digest respectively, so it should be easy to justify why you chose these values and put that in a comment.

Reply:
There isn't really a meaning.

It's the smallest size buffer that will hold all of the possible response data for the operations Tock currently supports. So as we support more operations it might continue to grow. The idea being to just limit the buffer size to as small as possible

Reply:
And you determine that empirically?

Reply:
Yes